### ISSUE TITLE
Create coil profile field mapping SOP

### ISSUE THREAD

**Issue by po09i** [shimming-toolbox/shimming-toolbox#488]
### Description
We should have a page in the documentation that includes some information about acquiring field maps with the purpose of creating coil profiles.We should add it near the coil profile tutorial (probably in a section above what is currently there)

Important information:
- Do a first shim at the start and do not change that shim for the remainder of the acquisitions (including not changing f0) 
  - The important part is that we do not want to adjust anything between scans of the same channel

- To speed things up and avoid human error, use an FID with trigger connected to the custom coil between each field-mapping sequence to change the shim (1 amp channel 1, -1 amp channel 1, ...) automatically

**Comment by jcohenadad** [comment]
> To speed things up and avoid human error, use an FID with trigger connected to the custom coil between each field-mapping sequence to change the shim (1 amp channel 1, -1 amp channel 1, ...) automatically

@jaystock we tried with the sequence `Siemens/ServiceSequences/DefaultProtocols/fidseq`, but we ended up with 39 TTLs. We made sure the number of average was 1 and steady-state stabilization was unchecked. Can you please let us know which sequence you used? Thanks!

**Comment by jaystock** [comment]
Yes, let me check on the scanner and export the protocol that was used.

Sent from my iPhone

On Oct 13, 2023, at 10:51 AM, Julien Cohen-Adad ***@***.***> wrote:

﻿

        External Email - Use Caution

To speed things up and avoid human error, use an FID with trigger connected to the custom coil between each field-mapping sequence to change the shim (1 amp channel 1, -1 amp channel 1, ...) automatically

@jaystock<https://secure-web.cisco.com/18mKoPMN0VhQCWsPUL8PgTZrhDUHMviQaXE3wDr7UaKZC6VEyYaPM8MD7JU5uhv9hOAIuNU80SGkA0LHBKEjF94eyW0mdsrTNF-aHM1Uy52yG2aypi2XoPBb5DXASz0fmoAOY1axoMxtJ1XAQWMcNgakjKD1W-sTx5IqDYIs-m5RcSUdBv-7yQY-nPhyqwY3HVbUI6jh2tZRkRIUCgLrPTqZodmdhjsjmqLC7rd3eV8LfMI3o4F2nC9okUZC4hYpYyR_C9A0b3QGdqDR82E4djYLy1KRF0I20jdU6DHYo7QyYAIF7tfOukxwjBl9SkKFP/https%3A%2F%2Fgithub.com%2Fjaystock> we tried with the sequence Siemens/ServiceSequences/DefaultProtocols/fidseq, but we ended up with 39 TTLs. We made sure the number of average was 1 and steady-state stabilization was unchecked. Can you please let us know which sequence you used? Thanks!

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/1MIooP68Wp76fvUvr0esNTgWk9nJ7cpHGm6YiJ8JIFtObg090hzo9DFYrXQfmM7icUmQCgNXl49dcHuXxH19wwKtvJGx9-OXZ90XkXmuE7RIFsmqAIdWqSQuvMiWglCnfd0ksXUMDS0rhfte6LtABKYpEXjGewNhZMF6Xu5VDkJ9Y4POODMzf1HowhvBbZbFDQiJxcDDVp87NO7J3gMeDNowaz7NcsVpdlWBsFulQQteB_Zo7pBBsCU44Ko830ZvesAGWM6lJJFCs_HIMttEsWeX2dNfj0pH0oM8HRAsK3Mjyln5NJkWeosPQx_fYZfFV/https%3A%2F%2Fgithub.com%2Fshimming-toolbox%2Fshimming-toolbox%2Fissues%2F488%23issuecomment-1761651136>, or unsubscribe<https://secure-web.cisco.com/13NDhX5cKbYiXdKGlTauQfTcjcZl0VAV0TvQQP76VNG_uobl-J4uNpoUWxbtatioNc4jffB8IycQRB-HBpOiHVgeT5QBW8qBOgWKrLaKsARQ8khJcpS_CJ_7iRaTT7Cd4zduNlHuf4KOoJJr7bjsc9W19x5gCG1aacMyv85oKwlHsa7rRsUQ1zEmMlnzqZEkkojbCWTQksy1cJJmunp6QTFb6rfrMj1xxQMwq7edR0jfeJi9L3LgT8EcByGawOfJY6AsFCsxiFTJwTRjXFpX-agovnNAkVaM9qX56yHD9tk-sJutLXeFHWJkH5yTBlDgk/https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAGCOLGZD33IZ7LJQZJZUDRDX7FIQTANCNFSM6AAAAAA5RISYEE>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
The information in this e-mail is intended only for the person to whom it is addressed.  If you believe this e-mail was sent to you in error and the e-mail contains patient information, please contact the Mass General Brigham Compliance HelpLine at https://www.massgeneralbrigham.org/complianceline <https://www.massgeneralbrigham.org/complianceline> .
Please note that this e-mail is not secure (encrypted).  If you do not wish to continue communication over unencrypted e-mail, please notify the sender of this message immediately.  Continuing to send or respond to e-mail after receiving this message means you understand and accept this risk and wish to continue to communicate over unencrypted e-mail.
